src/modules/universal-discovery/components/selected-content/selected.content.component.js

```diff
@@ -117,14 +117,18 @@ export default class SelectedContentComponent extends Component {
     }
 
     render() {
-        const titles = this.props.items.map((item) => item.ContentInfo.Content.Name).join(', ');
+        const { items } = this.props;
+        const titles = items.map((item) => item.ContentInfo.Content.Name).join(', ');
+        const anyItemSelected = !!items.length;
+        const cssClassOnAnyItemSelected = anyItemSelected ? 'c-selected-content--any-item-selected' : '';
+        const cssClasses = `c-selected-content ${cssClassOnAnyItemSelected}`;
```

Comment:
I would be better to use something like this: https://www.npmjs.com/package/classnames
What do you think, @sunpietro @dew326 ?